core/tests/puppeteer-acceptance-tests/spec/practice-question-submitter-tests/suggest-medium-drag-and-drop-sort-question.spec.js

```diff
@@ -0,0 +1,192 @@
+      { topicName, skillDescription });
+    await questionSubmitter.expectQuestionDifficultyChoices(
+      {
+        difficulty: DIFFICULTY_MEDIUM,
+        rubricNotes: mediumDifficultyRubricNotes
+      },
+      {
+        difficulty: DIFFICULTY_HARD,
+        rubricNotes: hardDifficultyRubricNotes
+      });
+
+    await questionSubmitter.chooseDifficulty(DIFFICULTY_MEDIUM);
+    await questionSubmitter.expectChosenDifficultyToBe(
+      DIFFICULTY_MEDIUM);
+    await questionSubmitter.expectSkillRubricNotesToHave(
+      mediumDifficultyRubricNotes);
+    await questionSubmitter.expectQuestionTextSection();
+
+    await questionSubmitter.writeQuestionText(questionText);
+    await questionSubmitter.expectQuestionTextToBe(questionText);
+
+    await questionSubmitter.addInteraction();
+    await questionSubmitter.expectInteractionChoices();
+    await questionSubmitter.navigateToCommonInteractions();
+    await questionSubmitter.chooseInteraction(
+      INTERACTION_DRAG_AND_DROP_SORT);
+    await questionSubmitter.expectDragAndDropSortInteractionEditor();
+    await (
+      questionSubmitter.expectUnmetMinimumDragAndDropSortItemCount());
+    await questionSubmitter.expectDragAndDropSortItemCountToBe(1);
+
+    await questionSubmitter.writeDragAndDropSortItems(['3', '1', '2']);
+    await questionSubmitter.expectDragAndDropSortItemCountToBe(3);
+    await questionSubmitter.allowMultipleSortItemsInSamePosition();
+    await questionSubmitter.saveInteraction();
+    await questionSubmitter.expectResponseEditor();
+
+    await questionSubmitter.chooseResponseAssertion(
+      RESPONSE_EQUAL_TO_ORDERING);
+    await questionSubmitter.writeResponseFeedback(correctAnswer);
+    await questionSubmitter.saveResponseAndAddAnother();
+    await questionSubmitter.expectResponseEditor();
+    await questionSubmitter.chooseResponseAssertion(
+      RESPONSE_EQUAL_TO_ORDERING_WITH_ONE_ITEM_AT_INCORRECT_POSITION);
+    await questionSubmitter.tagResponseWithMisconception(
+      misconception);
+    await questionSubmitter.saveResponse();
+    await questionSubmitter.expectResponseToExist({
+      assertion: RESPONSE_EQUAL_TO_ORDERING,
+      feedback: correctAnswer
+    });
+    await questionSubmitter.expectResponseToExist({
+      assertion: RESPONSE_EQUAL_TO_ORDERING_WITH_ONE_ITEM_AT_INCORRECT_POSITION,
+      feedback: misconception.feedback
+    });
+    await questionSubmitter.expectResponseToExist({
+      assertion: RESPONSE_ALL_OTHER_ANSWERS_ARE_WRONG,
+      feedback: ''
+    });
+    await questionSubmitter.addHint(hint);
+    await questionSubmitter.expectHint(hint);
+    await questionSubmitter.expectMissingSolution();
+    await questionSubmitter.addSolution({
+      quantity: SOLUTION_QUANTITY_ONLY_ONE, solution, explanation
+    });
+    await questionSubmitter.expectSolutionToExist({
+      solution, explanation });
+    await questionSubmitter.expectDefaultOutcomeWithNoFeedback();
```

Comment:
Not sure what's being expected here, since "default outcome with no feedback" is a noun. Perhaps this is supposed to be expectDefaultOutcomeHasNoFeedback? Or expectWarningShownForFeedbacklessDefaultOutcome? (Not sure, feel free to edit as needed.)

Reply:
The feature related to this review thread was removed.